jobs/build.Jenkinsfile

```diff
@@ -267,7 +221,7 @@ lock(resource: "build-${params.STREAM}") {
             def version = ""
             if (params.VERSION) {
                 version = "--version ${params.VERSION}"
-            } else if (official) {
+            } else {
                 def new_version = shwrapCapture("/var/tmp/fcos-releng/scripts/versionary.py")
```

Comment:
Hmm, I don't think we should encourage anyone to produce development artifacts versioned like official ones.  It seems likely to create confusion.

Reply:
That's fair. Added a new commit to address this concern.